scripts/gh-pr-assigned.coffee

```diff
@@ -1,7 +1,10 @@
 # Description:
-#   Notifies ram535ii when he has been assigned to a PR
+#   Notifies HUBOT_GITHUB_USERNAME when they have been assigned to a Pull Request
 #
 # Configuration:
+#   HUBOT_GITHUB_EVENT_NOTIFIER_ROOM  - The default room to which message should go (recommend private chat, i.e. your Slack username)
+#   HUBOT_GITHUB_USERNAME - The github user to check for pul request assignment
```

Comment:
@seddy I added in HUBOT since when you are looking in your Heroku environment variables it is clear that these are related directly to your HUBOT and not something else. Thoughts?

Also seems like it might be conventional, not sure, will check.